src/main/java/Station.java

```diff
@@ -0,0 +1,47 @@
+import java.util.List;
+
+public class Station
```

Comment:
I would not call it Station since the data inside it represents all the Stations in the system.